Extensions/ArtifactEngine/package.json

```diff
@@ -9,19 +9,21 @@
   },
```

Comment:
looks like i need to again register for the components we are using now :( are you sure these changes are required?